blog/tests/views_test.py

```diff
@@ -0,0 +1,115 @@
+"""Tests for views' Blog app."""
+
+from datetime import datetime
+
+from django.contrib.auth.models import User
+from django.test import TestCase
+from django.urls import reverse
+
+from ..models import Post
+
+
+class PostListTest(TestCase):
+    """Tests for Post model."""
+
+    def setUp(self):
+        """Set up non-modifiable objects used by all test methods."""
+        bob = User.objects.create(username='bob')
+        self.post_first = Post.objects.create(
+            author=bob,
+            title='Title Bob',
+            text='Text bob',
+            published_date=datetime(day=8, month=11, year=2018),
+        )
+
+        john = User.objects.create(username='john')
+
+        self.post_second = Post.objects.create(
+            author=john, title='Title John', text='Text John'
+        )
+        self.post_third = Post.objects.create(
+            author=john, title='Title John 2', text='Text John 2'
+        )
+        self.post_third.publish()
+
+    def tearDown(self):
+        """Clean-up test data."""
+        del self.post_first
```

Comment:
Why not clean up everything?

Reply:
Sorry @webknjaz but I clean all https://github.com/kpi-web-guild/django-girls-blog-serhii73/blob/feature/create_unit_tests/blog/tests/views_test.py#L35

Reply:
```
    def tearDown(self):
        """Clean-up test data."""
        del self.post_first
        del self.post_second
        del self.post_third
```